lldb/test/Shell/Expr/TestProcessModificationIdOnExpr.cpp

```diff
@@ -36,34 +39,49 @@ int main() {
 }
 
 // CHECK-LABEL: process status -d
-// CHECK: m_stop_id: 2
-// CHECK: m_memory_id: 0
+// CHECK: m_stop_id: [[#STOP_ID:]]
+// CHECK: m_memory_id: [[#MEMORY_ID:]]
 
 // CHECK-LABEL: expr x.i != 42
 // IDs are not changed when executing simple expressions
 
 // CHECK-LABEL: process status -d
-// CHECK: m_stop_id: 2
-// CHECK: m_memory_id: 0
+// CHECK: m_stop_id: [[#STOP_ID]]
+// CHECK: m_memory_id: [[#MEMORY_ID]]
+
+// CHECK-LABEL: process status -d
+// Remember new values
```

Comment:
If the IDs are not changed, why do you need to capture them again here?

Reply:
Sure, we can avoid recapturing them but I decided that it is more readable to use the same pattern (all 4 times):
1. Capture variables;
2. Do some "test" command;
3. Check the variables.

Otherwise the fact whether we need to capture variable for "testing" next command depends on what was done in the previous command. And some commands change one value but not the other and it becomes messy so I decided that it is simpler to always recapture them.

I can change that and avoid unnecessary recaptures if you think that it is bad idea.

Reply:
Oh I see what you mean. Otherwise you have to unravel the sequence to add anything new and we're liable to make a mistake.